src/tests/integration/test-data/show/AssertTest.test_failing_branch(uint256).expected

```diff
@@ -32,84 +32,114 @@
 ┃
-┃ (branch)
-┣━━┓ constraint: { true #Equals 100 <=Int VV0_x_114b9705:Int }
-┃  │
-┃  ├─ 9
-┃  │   k: JUMPI 1124 bool2Word ( 100 <=Int VV0_x_114b9705:Int ) ~> #pc [ JUMPI ] ~> #execu ...
-┃  │   pc: 1116
-┃  │   callDepth: 0
-┃  │   statusCode: STATUSCODE:StatusCode
-┃  │
-┃  │  (39 steps)
-┃  ├─ 11
-┃  │   k: #end EVMC_SUCCESS ~> #pc [ STOP ] ~> #execute ~> CONTINUATION:K
-┃  │   pc: 328
-┃  │   callDepth: 0
-┃  │   statusCode: STATUSCODE:StatusCode
+┃ (1 step)
+┣━━┓
 ┃  │
-┃  │  (1 step)
-┃  ├─ 13
-┃  │   k: #halt ~> #pc [ STOP ] ~> #execute ~> CONTINUATION:K
-┃  │   pc: 328
-┃  │   callDepth: 0
-┃  │   statusCode: EVMC_SUCCESS
-┃  │
-┃  │  (2 steps)
-┃  ├─ 15 (terminal)
+┃  ├─ 8 (terminal)
 ┃  │   k: #halt ~> CONTINUATION:K
-┃  │   pc: 328
-┃  │   callDepth: 0
-┃  │   statusCode: EVMC_SUCCESS
+┃  │   pc: ?_PC_CELL_5d410f2a:Int
+┃  │   callDepth: ?_CALLDEPTH_CELL_5d410f2a:Int
+┃  │   statusCode: ?STATUSCODE_FINAL:StatusCode
 ┃  │
-┃  ┊  constraint: true
+┃  ┊  constraint: OMITTED CONSTRAINT
 ┃  ┊  subst: OMITTED SUBST
 ┃  └─ 7 (leaf, target, terminal)
 ┃      k: #halt ~> CONTINUATION:K
 ┃      pc: PC_CELL_5d410f2a:Int
 ┃      callDepth: CALLDEPTH_CELL_5d410f2a:Int
 ┃      statusCode: STATUSCODE_FINAL:StatusCode
 ┃
-┗━━┓ constraint: { true #Equals ( notBool 100 <=Int VV0_x_114b9705:Int ) }
+┗━━┓
    │
-   ├─ 10
-   │   k: JUMPI 1124 bool2Word ( 100 <=Int VV0_x_114b9705:Int ) ~> #pc [ JUMPI ] ~> #execu ...
-   │   pc: 1116
+   ├─ 9
+   │   k: #next [ PUSH ( 1 ) ] ~> #execute ~> CONTINUATION:K
+   │   pc: 0
    │   callDepth: 0
    │   statusCode: STATUSCODE:StatusCode
    │
-   │  (63 steps)
-   ├─ 12
-   │   k: #end EVMC_REVERT ~> #pc [ REVERT ] ~> #execute ~> CONTINUATION:K
-   │   pc: 2995
+   │  (359 steps)
+   ├─ 10 (split)
+   │   k: JUMPI 1124 bool2Word ( 100 <=Int VV0_x_114b9705:Int ) ~> #pc [ JUMPI ] ~> #execu ...
+   │   pc: 1116
    │   callDepth: 0
    │   statusCode: STATUSCODE:StatusCode
-   │
-   │  (1 step)
-   ├─ 14
-   │   k: #halt ~> #pc [ REVERT ] ~> #execute ~> CONTINUATION:K
-   │   pc: 2995
-   │   callDepth: 0
-   │   statusCode: EVMC_REVERT
-   │
-   │  (2 steps)
-   └─ 16 (leaf, terminal)
-       k: #halt ~> CONTINUATION:K
-       pc: 2995
-       callDepth: 0
-       statusCode: EVMC_REVERT
+   ┃
+   ┃ (branch)
+   ┣━━┓ constraint: { true #Equals 100 <=Int VV0_x_114b9705:Int }
+   ┃  │
+   ┃  ├─ 11
+   ┃  │   k: JUMPI 1124 bool2Word ( 100 <=Int VV0_x_114b9705:Int ) ~> #pc [ JUMPI ] ~> #execu ...
+   ┃  │   pc: 1116
+   ┃  │   callDepth: 0
+   ┃  │   statusCode: STATUSCODE:StatusCode
+   ┃  │   src: /tmp/pytest-of-anvacaru/pytest-25/popen-gw0/foundry0/lib/forge-std/src/StdInvariant.sol:85:86
+   ┃  │
+   ┃  │  (39 steps)
+   ┃  ├─ 13
+   ┃  │   k: #end EVMC_SUCCESS ~> #pc [ STOP ] ~> #execute ~> CONTINUATION:K
+   ┃  │   pc: 328
+   ┃  │   callDepth: 0
+   ┃  │   statusCode: STATUSCODE:StatusCode
+   ┃  │   src: /tmp/pytest-of-anvacaru/pytest-25/popen-gw0/foundry0/lib/forge-std/src/StdInvariant.sol:69:70
```

Comment:
I am not sure what changed, but seems like these paths are causing the expected test to fail.